student.py

```diff
@@ -0,0 +1,8 @@
+class Student:
+    def __init__(self, id, name):
+        self.id = id
+        self.name = name
+        self.is_present = False
+
+    def mark_attendance(self):
+        self.is_present = True
```

Comment:
Consider adding a check to prevent marking attendance multiple times for the same student. This would help maintain accurate attendance data.